changelogs/current.yaml

```diff
@@ -79,6 +79,12 @@ minor_behavior_changes:
     Allow malformed URL encoded triplets in the default header validator. This behavior can be reverted by setting runtime flag
     ``envoy.reloadable_features.uhv_allow_malformed_url_encoding`` to false, in which case requests with malformed URL encoded triplets
     in path are rejected. This setting is only applicable when the Unversal Header Validator is enabled and has no effect otherwise.
+- area: aws
+  change: |
+    Default sanitize host header prior to direct invocation of a lambda.  This behavior can reverted by setting runtime guard
+    ``envoy.reloadable_features.lambda_sanitize_host_headers`` to false, in which case requests will retain the originating header to the
+    AWS invocation lambda API, causing a signature mismatch error to be returned.  This is only applicable when using
```

Comment:
`"causing a signature mismatch error to be returned"` Should we remove this?

I am unsure if this will return signature mismatch error on all scenarios. In some testing where the host header didn't contain the port number in the request host header the signature seem to be accepted.